mod-login-saml/src/main/resources/domain/mod-login-saml/features/samlCallback.feature

```diff
@@ -0,0 +1,25 @@
+# NOTE The saml/callback endpoint is called by the IdP after authentication is performed.
+# Because the saml/callback endpoint requires a signature from the IdP, doing a true integration test of it will
+# be difficult if not impossible.
+Feature: Test what we can for the callback endpoint
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers =
+    """
+    {
+      "X-Okapi-Tenant": "#(testTenant)",
+      "X-Okapi-Token": "#(okapitoken)"
+    }
+    """
+
+  Scenario: Test CORS for the callback endpoint
+    Given path "_/invoke/tenant/" + testTenant + "/saml/callback"
+    And header Origin = baseUrl
+    And header Access-Control-Request-Method = "POST"
+    When method OPTIONS
+    Then status 204
+    And match header access-control-allow-methods contains "POST"
+    And match header access-control-allow-origin == baseUrl
+    And match header access-control-allow-credentials == "true"
```

Comment:
@hjiebsco and @julianladisch I think this should take care of it. To verify that things were working as expected based on Hongwei's comments, I added a SAML binding manually on the testing reference environment by going to Settings > Tenants, and indeed the route that is called by the client JS, after logging in at the IdP, is ```_invoke/tenant/<tenant id>/saml/callback```, and the the new revision of the test passes.